### ISSUE TITLE
Instructions for ROV mode

### ISSUE THREAD

**Issue by samiamlabs** [clydemcqueen/orca4#17]
Hi!

I have been trying to figure out how the ROV mode is supposed to be used from reading the code.
My best guess is that it should be possible to connect something like qgroundcontrol with an xbox controller in parallel to the ROS2 system and have that take over when the manager is in ROV mode. Is that correct, or have I misunderstood something?

I think it would be nice if a short explanation of how ROV mode is intended to be used could be added to a README.

**Comment by clydemcqueen** [comment]
Yes, that's how I do it. I use QGC mainly to provide the joystick driver. QGC connects to mavros: https://github.com/clydemcqueen/orca4/blob/e12c4e1d75de37908f2be9b6d68627987ebc3fbf/orca_bringup/params/sim_mavros_params.yaml#L35

I'm happy for PRs!
